package.json

```diff
@@ -39,7 +39,7 @@
   "homepage": "https://github.com/mongoist/mongoist#readme",
   "dependencies": {
     "debug": "^4.1.1",
-    "mongodb": "^3.3.2"
+    "mongodb": "^4.8.0"
```

Comment:
The `engines` key in this file should be updated per https://github.com/mongodb/node-mongodb-native/blob/HEAD/etc/notes/CHANGES_4.0.0.md#nodejs-version. Perhaps copy https://github.com/mongodb/node-mongodb-native/blob/a983f147776e38a8c09a063120f64d5053619842/package.json#L82